server.js

```diff
@@ -7,6 +7,20 @@ const distPath = path.resolve(`${__dirname}/dist`);
 const port = process.env.PORT || 8080;
 const app = express();
 
+let sources = {
+  'default-src': ["'self'"],
```

Comment:
why is the double quotes necessary?

Reply:
its mandatory to have single quotes around the policy. https://developers.google.com/web/fundamentals/security/csp/ (search for 'These keywords require single-quotes').